pandas/io/excel/_base.py

```diff
@@ -155,9 +155,11 @@
     Returns a subset of the columns according to behavior above.
 dtype : Type name or dict of column -> type, default None
     Data type for data or columns. E.g. {{'a': np.float64, 'b': np.int32}}
-    Use `object` to preserve data as stored in Excel and not interpret dtype.
+    Use `object` to preserve data as stored in Excel and not interpret dtype,
+    which will necessarily result in `object` dtype.
     If converters are specified, they will be applied INSTEAD
     of dtype conversion.
+    If you use `None`, it will infer the dtype of each column based on the data.
```

Comment:
Not a big deal, especially because the issue existed prior to this PR, but I think we should be using double backticks here. The docstrings use RST format, not markdown, and so this renders as italics rather than code highlighting.

Reply:
Thanks for the feedback! I've sent a PR with the double backticks. Let me know if I can help with anything else!